modules/WebsiteCMS/WebsiteAboutUs/Handlers/UpdateWebsiteAboutUsHandler.php

```
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Modules\WebsiteCMS\WebsiteAboutUs\Handlers;
+
+use Modules\WebsiteCMS\WebsiteAboutUs\Commands\UpdateWebsiteAboutUsCommand;
+use Modules\WebsiteCMS\WebsiteAboutUs\Repositories\WebsiteAboutUsRepository;
+
+class UpdateWebsiteAboutUsHandler
+{
+    public function __construct(
+        private WebsiteAboutUsRepository $repository,
+    ) {
+    }
+
+    public function handle(UpdateWebsiteAboutUsCommand $updateWebsiteAboutUsCommand)
+    {
+        $this->repository->updateWebsiteAboutUs($updateWebsiteAboutUsCommand->getId(), $updateWebsiteAboutUsCommand->toArray());
```

Consider adding error handling around the repository call. What happens if the update fails? The method currently swallows all exceptions.